ksql-engine/src/main/java/io/confluent/ksql/util/PersistentQueryMetadata.java

```diff
@@ -45,6 +45,8 @@
   private final PhysicalSchema resultSchema;
   private final DataSourceType dataSourceType;
   private final Optional<MaterializationProvider> materializationProvider;
+  private Optional<Materialization> materialization;
+  private final QueryId queryId;
```

Comment:
we already have `QueryId id` - we should name this more descriptively. also it might be easier to read if we just created the queryID in the method instead of storing it as a field (esp. if it isn't used in hashCode/equals/toString).